### DISCUSSION TITLE
Error on install winter

### DISCUSSION THREAD

**Discussion by Tricky71** [wintercms/winter#180] [Q&A] [answered]
Got error after php artisan winter:install

[ERROR] exec(): Unable to fork [stty 2>&1]

 Database type [SQLite]:
  [0] MySQL
  [1] Postgres
  [2] SQLite
  [3] SQL Server
 >

Can you help, please?

**Comment by rubenvanerk** [comment]
Could you run `php -v` and post the output?

**Comment by bennothommo** [comment, marked as answer]
This error would occur in cases where you have some form of "shell fork bomb" protection - basically, it is preventing the install command from creating more system processes. You will need to disable this protection temporarily for the install command to work - if you are running cPanel and have access to the WHM panel, you can toggle this in the "Shell Fork Bomb Protection" screen.
